content/reference.html

```diff
@@ -247,15 +247,17 @@
 - The worker will do the actual work of building, compiling and testing the package and may optionally then upload the compiled package to Anaconda Cloud. 
 {% endcall %}
 
-{% call subsection('Channels')%}
+{% call subsection('Labels')%}
 The URLs on Anaconda Cloud where conda looks for packages. Using the Anaconda-Client 
-CLI, package developers can create additional channels such as development (channel/dev) 
-or test (channel/test) which will be searched only if the user specifies the channel.  
-
-https://anaconda.org/travis - the default channel  
-https://anaconda.org/travis/channel/main - same as the default channel  
-https://anaconda.org/travis/channel/dev - contains the packages in development  
-https://anaconda.org/travis/channel/test - contains packages ready to test.  
+CLI, package developers can create additional labels such as development (label/dev) 
+test (label/test) or other labels which will be searched 
+only if the user specifies the label.  
+
+https://anaconda.org/travis/label/main - the label searched by default
+https://anaconda.org/travis - same as default label with "main" implicit
+https://anaconda.org/travis/label/dev - contains the packages in development  
+https://anaconda.org/travis/label/test - contains packages ready to test
+https://anaconda.org/travis/label/any-custom-label - any label you wish to use.
```

Comment:
these URLs should say `labels` plural (i.e. https://anaconda.org/travis/labels/dev) - the original documentation was also incorrect.